poi-radio/src/config.rs

```diff
@@ -372,13 +372,11 @@ impl Config {
         .map_err(|e| QueryError::Other(e.into()))?;
         // The query here must be Ok but so it is okay to panic here
         // Alternatively, make validate_set_up return wallet, address, and stake
-        let my_address =
-            query_registry(self.registry_subgraph.to_string(), wallet_address(&wallet)).await?;
-        let my_stake =
-            query_network_subgraph(self.network_subgraph.to_string(), my_address.clone())
-                .await
-                .unwrap()
-                .indexer_stake();
+        let my_address = query_registry(&self.registry_subgraph, &wallet_address(&wallet)).await?;
```

Comment:
does it work well to let wallet_address return &str

Reply:
I tried it but after a lot of fiddling around, seems like returning `String` is the best approach
> In Rust, returning a String from a function that generates new string data is idiomatic because it adheres to the language's ownership model and guarantees memory safety, while returning a &str is typically not feasible as it requires the string data to live longer than the function scope, which isn't possible for strings generated within the function.